docs/Readme.md

```diff
@@ -1 +0,0 @@
-The documentation of the Logging operator project is available at the [Banzai Cloud Documentation Page](https://banzaicloud.com/docs/one-eye/logging-operator/).
```

Comment:
[optional]
I am fine with deleting it from there as the new docs are mentioned in the root README already, just wanted to ask why not change it to the newer one?

Reply:
I'm going to provide a short description on what kind of docs are here instead of linking to the site